pyensign/ensign.py

```diff
@@ -48,6 +49,18 @@ def __init__(
             Set to True to disable topic ID caching.
         """
 
+        try:
+            with open(cred_path, "r") as file:
+                data = json.load(file)  # Process the loaded JSON data here
+                client_id = data["ClientID"]
+                client_secret = data["ClientSecret"]
+        except FileNotFoundError:  # If the file is not found,
+            print("File not found. Please check ")
```

Comment:
I like that we're handling the different exceptions and providing meaningful errors to the user. To take it a step farther, can we raise `ValueError` for these instead of printing? Users will most likely be calling this code from an environment where they can't see the `stdout` output (e.g. a Docker container), but exceptions will be more noticeable since the code is interrupted.

```python
except FileNotFoundError as e:
    raise ValueError(f"cred_path not found: {cred_path}") from e
```